libc/docs/c23.rst

```diff
@@ -18,6 +18,9 @@ Implementation Status
 
 Additions:
 
+* setjmp.h
+
+  * longjmp |check|
```

Comment:
longjmp has been around since the early pre-dawn of C; this documentation page is for nascent C23 additions. Remove this hunk?